tools/witx/src/docs/ast.rs

```diff
@@ -184,7 +193,7 @@ impl ToMarkdown for StructDatatype {
                 MdHeading::new_bullet(),
                 id.as_str(),
                 name,
-                &member.docs,
+                format!("{}Offset: {}", &member.docs, &offset).as_str(),
```

Comment:
Would it make sense to add a newline after the offset here? Before this change, it looks like there's a blank line at that point, which creates a nice visual separation of different documentation items.